### DISCUSSION TITLE
Setting IP address when running prisma client locally

### DISCUSSION THREAD

**Discussion by schalkneethling** [prisma/prisma#14041] [Q&A] [answered]
When running the Prisma client locally and connecting to a MongoDB server on MongoDB Atlas how does one set the client to connect from say, `127.0.0.1`? Currently, I need to allow connections from anywhere temporarily in order for it to connect and query the database.

This is not ideal and definitely not an option when deploying to production. I am new to using Prisma so, I am probably overlooking something obvious :) Your help would be much appreciated. Thanks!

**Comment by austincrim** [comment, marked as answer]
Hey @schalkneethling 👋,

I might be misunderstanding your question, but the Prisma Client will connect from wherever your server is running. 

I typically don't need to do any further configuration in Mongo Atlas when using Prisma, what kind of settings do you need to tweak?

**Reply by schalkneethling** [reply]
Hi @austincrim, thank you for your feedback. When running locally via a Node express app I find that the Prisma client fails to connect even though I am allowing connections from `127.0.0.1` on Atlas. The only way I have been able to work around this at the moment is to allow connections from anywhere for a few hours. This is against a staging database so it is kinda OK for now :)

I am curious what would happen once I deploy this to a staging server on DigitalOcean for example though.

**Reply by austincrim** [reply]
I believe if you go to the Mongo Atlas dashboard and try to add another IP to the allowlist, it has a button for `Add My Current IP`. You could try that and if errors persist, we can open a bug report.

**Reply by schalkneethling** [reply]
Thank you, I will give this a try and let you know.

**Reply by schalkneethling** [reply]
This seems to have worked for me o/\o - I will probably have to update it as my IP is not static, but it is better than opening up for the world :) Once deployed on a server, _it_ will obviously have a static IP, so there will be no need to update it. Thanks, @austincrim

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
